content/gui/xml/ingame/widgets/captains_log.xml

```diff
@@ -22,10 +23,12 @@
 	</VBox> <!-- closes VBox logbook left page -->
 
 	<!-- right page -->
-	<VBox max_size="350,508" min_size="350,508" padding="5" name="logbook_right">
-		<VBox name="custom_widgets_right" max_size="350,508" min_size="350,0"
-			comment="In here, widgets defined in yaml get added on runtime." />
-	</VBox> <!-- closes VBox logbook right page -->
+    <VBox max_size="350,508" min_size="350,508" padding="5" name="logbook_right">
```

Comment:
Well, I can see it:
![2016-10-16-175751_749x312_scrot](https://cloud.githubusercontent.com/assets/697766/19418716/1f18dcaa-93ca-11e6-941e-d78746a66f44.png)
It's a tab vs. spaces thing.